lua/lualine/components/filename.lua

```diff
@@ -50,6 +50,19 @@ local function shorten_path(path, sep, max_len)
   return table.concat(segments, sep)
 end
 
+local function filename_and_parent(path)
+  local segments = vim.split(path, '/')
```

Comment:
on windows path separator would be forword slash right?

Reply:
You're right, the separator shouldn't be hardcoded. I think using `path_separator` which is already being set for `shorten_path` could be the smart move here. Will look at it soon.

Reply:
Can't test on Windows, but now uses `path_separator` like `shorten_path`.